Demo/src/main/java/com/braintreepayments/demo/Settings.java

```diff
@@ -27,6 +27,9 @@ public class Settings {
 
     static final String LOCAL_PAYMENTS_TOKENIZATION_KEY = "sandbox_f252zhq7_hh4cpc39zq4rgjcg";
 
+    private static final String XO_SANDBOX_TOKENIZATION_KEY = "sandbox_rz48bqvw_jcyycfw6f9j4nj9c";
```

Comment:
Are we able to use our default sandbox key that works with our sample merchant server?

Reply:
I think we needed to use a specific sandbox key initially for testing. @agedd - does iOS use the default sandbox key for the MPE flow?

Reply:
hmm i think i've only ever tested MPE w/ the clientToken auth flow, but i think these changes may be related to https://github.com/braintree/braintree_android/pull/996 and https://github.com/braintree/braintree_ios/pull/1298 where it looks like we're adding this new key to test the web checkout flow 👀

Reply:
Ah that's right! I'm not sure if our default sandbox key triggers the new web checkout flow. We should be safe to merge this in as is since the keys are determined by a setting preference.

Reply:
Did we check our default sandbox key? At one point in time I think it was supposed to be ramped for this flow but don't know if that was ever completed. It's fine as is if this is the only way to see that flow in sandbox.

Reply:
I just did a quick test, I'm seeing the old flow for one-time checkout for the default sandbox key. So we'll need to keep this in.